content/docs/command-reference/import.md

```diff
@@ -104,15 +104,18 @@ To actually [version the data](/doc/start/data-and-model-versioning), `git add`
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
+- `--no-exec` - create the import `.dvc` file without accessing `url` (assumes
   that the data source is valid). This is useful if you need to define the
   project imports quickly, and import the data later (use `dvc update` to finish
   the operation(s)).
 
-- `--no-download` - create the import `.dvc` with data checksums but without
-  downloading the associated data. This is useful if you need track changes in
-  remote data but do not (yet) need to download data to the local workspace.
-  Data can be later downloaded using `dvc pull`.
+- `--no-download` - create the import `.dvc` file including
+  [hash values](/doc/user-guide/project-structure/dvc-files#dependency-entries)
+  for the remote file but without downloading the associated data. This is
```

Comment:
Wait. There are no hash values in `deps` for `import` .dvc files, are there? It tracks a `repo` + `rev_lock` last time I checked.

Reply:
Good catch! You're right, there's no hash values in deps for `import --no-download`, which means that the dependencies aren't actually locked. Will need to fix this in `dvc` too. c.c. @dberenbaum

Reply:
Yup, great catch @jorgeorpinel!

Reply:
Fixed with https://github.com/iterative/dvc/pull/8341. Will push an update here soon.